src/component/strategies/StrategyForm/StrategyForm.tsx

```diff
@@ -1,194 +1,111 @@
                     error={Boolean(errors.name)}
-                    helperText={errors.name}
-                    onChange={e => {
-                        clearErrors();
-                        setName(trim(e.target.value));
-                    }}
-                    value={name}
-                    variant="outlined"
-                    size="small"
+                    errorText={errors.name}
+                    onFocus={() => clearErrors()}
                 />
-
-                <TextField
-                    className={commonStyles.fullwidth}
-                    multiline
+                <p className={styles.inputDescription}>
+                    What is your strategy description?
+                </p>
+                <Input
+                    className={styles.input}
+                    label="Strategy description"
+                    value={strategyDesc}
+                    onChange={e => setStrategyDesc(e.target.value)}
                     rows={2}
-                    label="Description"
-                    name="description"
-                    placeholder=""
-                    onChange={e => setDescription(e.target.value)}
-                    value={description}
-                    variant="outlined"
-                    size="small"
+                    multiline
                 />
 
                 <StrategyParameters
                     input={params}
                     count={params.length}
                     updateParameter={updateParameter}
+                    setParams={setParams}
+                    errors={errors}
                 />
                 <Button
                     onClick={e => {
                         e.preventDefault();
                         appParameter();
                     }}
+                    variant="outlined"
+                    color="secondary"
+                    className={styles.paramButton}
                     startIcon={<Add />}
                 >
                     Add parameter
                 </Button>
-
-                <ConditionallyRender
-                    condition={editMode}
-                    show={
-                        <Button
-                            type="submit"
-                            variant="contained"
-                            color="primary"
-                            style={{ display: 'block' }}
-                        >
-                            Update
-                        </Button>
-                    }
-                    elseShow={
-                        <FormButtons
-                            submitText={'Create'}
-                            onCancel={handleCancel}
-                        />
-                    }
-                />
-            </form>
-        </PageContent>
+            </div>
+            <div className={styles.buttonContainer}>
+                {children}
+                <Button onClick={handleCancel} className={styles.cancelButton}>
```

Comment:
Should add a `type="button"` here, to make sure this button won't submit the form.